src/screens/augmentedReality/ARShowScreen.js

```diff
@@ -180,11 +163,9 @@ const AugmentedRealityView = ({ sceneNavigator }) => {
         source={{ uri: object.vrx }}
         resources={[{ uri: object.png }]}
         type="VRX"
-        position={position}
+        position={[0, -1, -5]}
         scale={[0.02, 0.02, 0.02]}
-        rotation={rotation}
-        onDrag={moveObject}
-        onRotate={rotateObject}
+        rotation={[0, 0, 0]}
```

Comment:
please keep these two as variables outside of the rendering, even if they are no state. we know that we will probably make these values coming from other data depending on the ar model:

```
const position = [0, -1, -5];
const rotation = [0, 0, 0];

...

position={position}
rotation={rotation}
```

and i think we should do it with the `scale` as well:

```
const scale = [0.02, 0.02, 0.02];

...

scale={scale}
```

Reply:
https://github.com/ikuseiGmbH/smart-village-app-app/pull/584/commits/c66b97c24afe2ee31d4e1c6d02332677c239d827

Reply:
nice